multinet/db.py

```diff
@@ -132,358 +107,33 @@ def workspace_mapping(name: str) -> Optional[Dict]:
-    RETURN LENGTH({table})
-    """
-    return next(aql_query(workspace, count_query))
-
-
-def workspace_table_keys(
-    workspace: str, table: str, filter_keys: bool = False
-) -> List[str]:
-    """Get the keys of a table in a workspace."""
-
-    query = f"""
-    FOR d in {table}
-        LIMIT 0, 1
-        RETURN ATTRIBUTES(d)
-    """
-    cur = aql_query(workspace, query)
-
-    try:
-        keys = next(cur)
-    except StopIteration:
-        return []
-
-    if filter_keys:
-        return [k for k in keys if k not in restricted_keys]
-
-    return keys
-
-
-def create_aql_table(workspace: str, name: str, aql: str) -> str:
-    """Create a new table from an AQL query."""
-    db = get_workspace_db(workspace, readonly=True)
-
-    if db.has_collection(name):
-        raise AlreadyExists("table", name)
-
-    # In the future, the result of this validation can be
-    # used to determine dependencies in virtual tables
-    rows = list(_run_aql_query(db.aql, aql))
-    validate_csv(rows, "_key", False)
-
-    db = get_workspace_db(workspace, readonly=False)
-    coll = db.create_collection(name, sync=True)
-    coll.insert_many(rows)
-
-    return name
-
-
-def graph_node(workspace: str, graph: str, table: str, node: str) -> dict:
-    """Return the data associated with a particular node in a graph."""
-    space = get_workspace_db(workspace)
-    graphs = filter(lambda g: g["name"] == graph, space.graphs())
-    try:
-        next(graphs)
-    except StopIteration:
-        raise GraphNotFound(workspace, graph)
-
-    tables = filter(lambda t: t["name"] == table, space.collections())
-    try:
-        next(tables)
-    except StopIteration:
-        raise TableNotFound(workspace, table)
-
-    query = f"""
-    FOR d in {table}
-      FILTER d._id == "{table}/{node}"
-      RETURN d
-    """
-
-    result = aql_query(workspace, query)
-    try:
-        data = next(result)
-    except StopIteration:
-        raise NodeNotFound(table, node)
-
-    return {k: data[k] for k in data if k != "_rev"}
-
-
-def workspace_graphs(workspace: str) -> List[str]:
-    """Return a list of all graph names in workspace `workspace`."""
-    space = get_workspace_db(workspace)
-    return [graph["name"] for graph in space.graphs()]
+def user_collection() -> StandardCollection:
+    """Return the collection that contains user documents."""
+    sysdb = system_db(readonly=False)
 
+    if not sysdb.has_collection("users"):
+        sysdb.create_collection("users")
 
-def workspace_graph(workspace: str, graph: str) -> GraphSpec:
-    """Return a specific graph named `name` in workspace `workspace`."""
-    get_graph_collection(workspace, graph)
+    return sysdb.collection("users")
 
-    # Get the lists of node and edge tables.
-    node_tables = graph_node_tables(workspace, graph)
-    edge_table = graph_edge_table(workspace, graph)
 
-    return {"nodeTables": node_tables, "edgeTable": edge_table}
+def search_user(query: str) -> Cursor:
```

Comment:
Why can't this be a static method of the `User` class?

Reply:
The main reason I didn't include it in that class is that it doesn't really operate on a specific user, or interact with the `User` class in anyway, like most of the other static methods do. I understand that it would be good to have for organization though, so I can move it.